### PR TITLE
Webpack 4

### PR DESCRIPTION
**Pull request by langpavel**
Big thanks to @rodeyseijkens

Please enjoy, test, see #1562

### CODE REVIEW

**Comment by langpavel** [comment]
This branch is support for #1562, if you have questions, please go there…

**Comment by langpavel** [comment]
I'm going to merge this stuff into `feature/redux`, other branches will follow…

File: tools/webpack.config.js [hunk 1]
```diff
@@ -331,8 +336,8 @@ const clientConfig = {
       cacheGroups: {
         commons: {
           chunks: 'initial',
-          test: /[\\/]node_modules[\\/]/,
-          name: 'vendor',
+          test: /node_modules/,
```
Comment (frenzzy) [inline tools/webpack.config.js]: JFYI, previously something like `bla_node_modules_bla` did no match.

Comment (langpavel) [reply tools/webpack.config.js]: reverted :+1:

**Pull request merged**